build error on linux: missing colordlg.h

I'm trying to compile this on linux. I get this build error:

```
~/convolutional_encoder_viterbi_decoder/encode.c:3:10: fatal error: colordlg.h: No such file or directory
    3 | #include "colordlg.h"
      |          ^~~~~~~~~~~~
compilation terminated.
```

Looks like it's a windows specific header. Does this code work on linux?

--- a/encode.c
+++ b/encode.c
@@ -1,6 +1,5 @@
 #include "fec.h"
 #include "colors.h"
-#include "colordlg.h"
 #define CUR_FILE "ENCODE.C"
 
 size_t encode(FEC _fec, uint8_t *msg, size_t sz_msg, uint8_t *enc_msg)

